govcd/lbappprofile_test.go

```diff
@@ -27,28 +27,28 @@ func (vcd *TestVCD) Test_LBAppProfile(check *C) {
 	check.Assert(edge.EdgeGateway.Name, Equals, vcd.config.VCD.EdgeGateway)
 
 	// Used for creating
-	lbAppProfileConfig := &types.LBAppProfile{
-		Name: TestLBAppProfile,
-		Persistence: &types.LBAppProfilePersistence{
+	lbAppProfileConfig := &types.LbAppProfile{
+		Name: TestLbAppProfile,
+		Persistence: &types.LbAppProfilePersistence{
 			Method: "sourceip",
 			Expire: 13,
 		},
 		Template: "HTTPS",
 	}
 
-	createdLbAppProfile, err := edge.CreateLBAppProfile(lbAppProfileConfig)
+	createdLbAppProfile, err := edge.CreateLbAppProfile(lbAppProfileConfig)
 	check.Assert(err, IsNil)
 	check.Assert(createdLbAppProfile.ID, Not(IsNil))
 
 	// We created application profile successfully therefore let's add it to cleanup list
 	parentEntity := vcd.org.Org.Name + "|" + vcd.vdc.Vdc.Name + "|" + vcd.config.VCD.EdgeGateway
-	AddToCleanupList(TestLBAppProfile, "lbAppProfile", parentEntity, check.TestName())
+	AddToCleanupList(TestLbAppProfile, "lbAppProfile", parentEntity, check.TestName())
 
 	// Lookup by both name and ID and compare that these are equal values
-	lbAppProfileByID, err := edge.ReadLBAppProfile(&types.LBAppProfile{ID: createdLbAppProfile.ID})
+	lbAppProfileByID, err := edge.GetLbAppProfile(&types.LbAppProfile{ID: createdLbAppProfile.ID})
 	check.Assert(err, IsNil)
 
-	lbPoolByName, err := edge.ReadLBAppProfile(&types.LBAppProfile{Name: createdLbAppProfile.Name})
+	lbPoolByName, err := edge.GetLbAppProfile(&types.LbAppProfile{Name: createdLbAppProfile.Name})
 	check.Assert(err, IsNil)
 	check.Assert(createdLbAppProfile.ID, Equals, lbPoolByName.ID)
```

Comment:
There should be a check that `lbPoolByName` is not nil

Reply:
Added here, and to more places.